app/src/main/res/layout-land/state_fragment.xml

```diff
@@ -81,6 +81,11 @@
           android:layout_height="wrap_content"
           android:visibility="@{viewModel.isAudioBarVisible ? View.VISIBLE : View.GONE}" />
       </FrameLayout>
+
+      <nl.dionsegijn.konfetti.KonfettiView
+        android:id="@+id/full_screen_confetti_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
```

Comment:
Update this to below code so that constraints can be used correctly. Ditto in other files.
```
        android:layout_width="0dp"
        android:layout_height="0dp"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toTopOf="parent" />
```

Reply:
Done

Reply:
0dp is conventional when we're relying on constraint layout to compute the value (it actually has a special meaning for constraint layout: https://developer.android.com/reference/androidx/constraintlayout/widget/ConstraintLayout#widgets-dimension-constraints).

Reply:
Agreed. For ConstraintLayout `0dp` is more correct as compared to `match_parent`.
Resolving this now.